content/katas/string-rotate/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: String rotate
+pre: "<b>4. </b>"
+weight: 5
+---
+
+
+## String rotate forward/backwards
+
+Write a function that takes in string argument and n (number) argument and the rotate the given string to nth position
+
+```
+//javascript
+rotate("hello",1) => // Expected output: elloh
+rotate("hello",2) => // Expected output: llohe
+rotate("hello",3) => // Expected output: lohel
+
+rotate("hello",-1) => // Expected output: ohell
+rotate("hello",-2) => // Expected output: lohel
+rotate("hello",-3) => // Expected output: llohe
+```
+
+```
+//py
+def rotate("hello",1)  # Expected output: elloh
+def rotate("hello",2)  # Expected output: llohe
+def rotate("hello",3)  # Expected output: lohel
+
+def rotate("hello",-1):  # Expected output: ohell
+def rotate("hello",-2):  # Expected output: lohel
+def rotate("hello",-3):  # Expected output: llohe
+```
```

Comment:
![ElijahCodeRev](https://user-images.githubusercontent.com/46021425/69326653-d47d7780-0c54-11ea-9c30-8763854c4a80.png)

You have a blank webpage. Add some information on it.